src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -0,0 +1,300 @@
+            new ActionArgument("abandonedSinceTimeInHours", ActionArgumentType.INTEGER, ActionArgumentType.Is.Optional)
+            )),
+    RequestSystemResetAll(new ActionMenu(
+            "RequestSystemResetAll",
+            "rsResetAll",
+            ForgePermissionNodes.RequestSystemResetAll,
+            RSResetAllCommand.class
+            )),
+
+    ListCitizens(new ActionMenu(
+            "ListCitizens",
+            "list",
+            ForgePermissionNodes.ListCitizens,
+            ListCitizensCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("page", ActionArgumentType.INTEGER, ActionArgumentType.Is.Optional)
+        )),
+    KillCitizen(new ActionMenu(
+            "KillCitizen",
+            "kill",
+            ForgePermissionNodes.KillCitizen,
+            KillCitizenCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required,
+                    new ActionArgument("citizen", ActionArgumentType.Citizen, ActionArgumentType.Is.Required))
+        )),
+    RespawnCitizen(new ActionMenu(
+            "RespawnCitizen",
+            "respawn",
+            ForgePermissionNodes.RespawnCitizen,
+            RespawnCitizenCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required,
+                    new ActionArgument("citizen", ActionArgumentType.Citizen, ActionArgumentType.Is.Required))
+        )),
+    CitizenInfo(new ActionMenu(
+            "Info",
+            "info",
+            ForgePermissionNodes.CitizenInfo,
+            CitizenInfoCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required,
+                    new ActionArgument("citizen", ActionArgumentType.Citizen, ActionArgumentType.Is.Required))
+        )),
+
+    ShowColonyInfo(new ActionMenu(
+            "ShowColonyInfo",
+            "info",
+            ForgePermissionNodes.ShowColonyInfo,
+            ShowColonyInfoCommand.class,
+            // TODO: need OR() ActionArgumentType
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Optional),
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Optional)
+            )),
+    DeleteColony(new ActionMenu(
+            "DeleteColony",
+            "delete",
+            ForgePermissionNodes.DeleteColony,
+            DeleteColonyCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("canDestroy", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Optional),
+            new ActionArgument("confirmDelete", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Optional)
+            )),
+    DisableBarbarianSpawns(new ActionMenu(
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00115)